tests/Integration/Behaviour/Features/Scenario/Product/Image/update_image.feature

```diff
@@ -104,6 +108,7 @@ Feature: Update product image from Back Office (BO)
       | image4          | false    |               | 4        | http://myshop.com/img/p/{image4}.jpg | http://myshop.com/img/p/{image4}-small_default.jpg |
       | image5          | false    |               | 5        | http://myshop.com/img/p/{image5}.jpg | http://myshop.com/img/p/{image5}-small_default.jpg |
       | image2          | false    |               | 6        | http://myshop.com/img/p/{image2}.jpg | http://myshop.com/img/p/{image2}-small_default.jpg |
+    And product "product1" should have following cover "http://myshop.com/img/p/{image1}.jpg"
```

Comment:
I didn't notice this before, in this tab, do we load an original image? maybe it would be better to load some type of image? to decrease load time

Reply:
@kpodemski Well, that's the only one we know for sure that it exists... Some thumbnail would be better, but you need to make sure:
A) Its generated. (Not sure if it is fixed already, on-the-fly generation didn't work for PNGs)
B) This thumbnail size is registered and used for products. 

Try to think with the team if it wouldn't be good to make some image sizes undeletable, like some order statuses are. At least the ones used in BO should be like this.

Reply:
Theoretically, all thumbnails are indeed present and available, as long as the image type has not been removed. We are indeed thinking of preventing the removal of basic image types, but that's another topic.

In this case the image is used in the new Product form where we display the cover in the header to help the merchant identify immediately which product he is editing:

<img width="1229" alt="Capture d’écran 2021-07-26 à 11 09 19" src="https://user-images.githubusercontent.com/13801017/126964131-312c6d3a-82d3-413b-9827-3dba28377743.png">

The graphical display is not definitive as for now we only have some functional mock-ups available, but that's the generic idea.

Reply:
So I don't know which image type I should use, I thought that by default I could use the original one to be sure the quality is good, but maybe it would be better to use a thumbnail. Since it's displayed in the BO I'm not sure the download cost is that heavy, but we can always modify this later anyway.

That said if you think it's preferable to get the thumbnail right now I can do the modif, using the `small_default` type?